TestTracer/Program.cs

```diff
@@ -1,28 +1,39 @@
 ﻿using System;
-using System.Collections.Generic;
 using System.IO;
-using System.Reflection;
+using System.Linq;
 using System.Threading;
-using FormatterContract;
 using TracerLib.TracerImpl;
-using  Result;
+using Result;
 using TracerLib.Model;
 
 
 namespace TestTracer
 {
-    class Program
+    public class Program
     {
         private static Tracer tracer = Tracer.GetInstance();
 
-        [STAThread]
         static void Main(string[] args)
         {
+            bool run = true;
+            bool first = true;
             var p = new Program();
             p.RunTestMethods();
```

Comment:
Let user to know that program done something and ready for input.

Reply:
I didn't  do it, because  in the task to run program we have to use this as parameters in command prompt.